components/Snackbar/snackbar.css

```diff
@@ -0,0 +1,128 @@
+
+:root {
+  --primary-color: #ff7474;
+  --primary-text-color: #ff5454;
+  --background-color: #fde3e3;
+  --border-color: #d3d3d3;
+}
```

Comment:
better to keep all variable items in one CSS file

Reply:
okay